config/default.js

```diff
@@ -54,4 +54,6 @@ module.exports = {
     }
   },
   mssql: mssql(),
+  sessionSecret: 'replace with unique session secret',
```

Comment:
Is there any good way to autogenerate this?  That would improve the DX of this sort of thing nicely.

Reply:
It's a great idea to do it, but very low value at the moment given that every "how-to" ever says to just generate your own, which we can tell users to do also. A good incremental improvement would be to read it from an env var or set to a constant, so there's no code modification required.